src/plugins/core.js

```diff
@@ -380,12 +383,24 @@ function Plugin(options = {}) {
     const transform = state.transform()
 
     if (isInternal) transform.delete()
-
-    return transform
-      .select(target)
-      .insertBlock(node)
-      .removeNodeByKey(node.key)
-      .apply()
+    if (isInline && !node.isVoid) return
+
+    switch (true) {
```

Comment:
Hey @YurkaninRyan can you switch this to not use `switch` and instead use a normal `if`? Thanks!